scheduler/server/stateful_scheduler.go

```diff
@@ -1176,7 +1189,9 @@ func (s *statefulScheduler) SetSchedulerStatus(maxTasks int) error {
 	if err != nil {
 		return err
 	}
-	s.config.TaskThrottle = maxTasks
+	s.setThrottle(maxTasks)
```

Comment:
Got confused here again. These functions for GetSchedulerStatus and SetSchedulerStatus sound generic but are only used for the throttle setting. Can we rename or is that going to break APIs (manual only APIs may be OK to break).

Reply:
I that the name is not good... 
Sickle is currently using it to select the cluster.

IMO I should merge the alg params into this.... 
- add a --throttle option, but also keep the current behavior, then
- Update sickle to use the --throttle option, then
- remove the default throttle behavior.

Reply:
So we'll end up with one pair of get/set settings APIs for all settings including throttle and algorithm? And we'll use flags for the CLI tools to make shortcuts? (SGTM)

Reply:
this is going to be a larger change, will create ticket for it.

Reply:
see MGCI-1310